jenkinsbuilder/weekly-scan.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+import beanstalkc
+import json
+import os
+import subprocess
+import sys
+import yaml
+
+
+# connect to beanstalkd tube
+bs = beanstalkc.Connection("BEANSTALK_SERVER")
+bs.use("master_tube")
+
+# registry server value to be replaced by ansible
+registry = "JENKINS_SLAVE"
+
+# cli argument pointing to directory containing yml files
+index_dir = sys.argv[1]
+
+# string representation of the catalog on r.c.o
+str_catalog = subprocess.check_output(
+    ["curl", "%s:5000/v2/_catalog" % registry]
+)
+
+# convert string into a list to use for comparison
+json_catalog = json.loads(str_catalog).values()[0]
+
+# have a list of files in the index_dir
+for directory, dirnames, filenames in os.walk(index_dir):
+    files = filenames
```

Comment:
Hmm, `os.listdir()` seems to be a better choice to me. You have any other suggestions?